### ISSUE TITLE
Adopt `accessibilityHelpContent` in other views

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#210257]
Once the API is stable, we should adopt `accessibilityHelpContent` in other views, like the [SCM view](https://github.com/microsoft/vscode/issues/203577).

**Comment by meganrogge** [comment]
@jooyoungseo can you think of any views that would benefit from help dialogs?

**Comment by jooyoungseo** [comment]
@meganrogge  Maybe, debugger? I know there are a bunch of barriers in debugger. We can talk about it later.

**Comment by meganrogge** [comment]
Also should have this for views besides the debug console related to debugging
